metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -157,11 +158,19 @@ private boolean isPotentialDoiProperty(OntologyProperty ontologyProperty) {
 
     private OntologyProperty getMappedOntologyProperty(Statement statement) {
         String property = statement.getPredicate().getLocalName();
-        Optional<Citation> citationValue = Citation.getByProperty(property);
+        Optional<Citation> citationValue = Citation.getTagByString(property);
         Optional<DcTerms> dcTermsValue = DcTerms.getTermByValue(property);
```

Comment:
You've started calling this string tagName elsewhere, the argument property should probably be changed to reflect that.

Also, in general, the Citation and DcTerms classes are basically the same; certainly the methods getTagByString and getTermByValue are. Having them named the same would be nice, but perhaps we should go a step further and either make a general class for the both of them, or have a general MetaTag (or whatever) for what these classes have in common and then extend this class for each of these metadata tag types.